tests/mocks/state.ts

```diff
@@ -59,7 +59,7 @@ export class StateMock {
   public electronMirror = ELECTRON_MIRROR;
   public isBisectCommandShowing = false;
 
-  public Bisector = new BisectorMock();
+  public Bisector: BisectorMock | undefined = new BisectorMock();
```

Comment:
Under what circumstances will undefined be returned here?

Reply:
`Bisector` is undefined if there is no active bisect running. This is tested under the `'renders bisect dialog button if no bisect instance'` test!